device-registry/src/main/resources/db/migration/V5__system_info.sql

```diff
@@ -0,0 +1,7 @@
+
+CREATE TABLE DeviceSystem (
+  uuid char(36) NOT NULL,
+  system_info LONGTEXT NOT NULL,
```

Comment:
So currently we don't really know exactly how the json object should look like, it should contain information about the hardware and probably the manifest etc. The idea was that for the time being we just store a json object without knowing much more, and later we will specify more information when we know more precisely what we should store.

Do you think it is worth the effort of going with dynamic columns for this?